mount/mount_idmapped_linux.go

```
@@ -0,0 +1,166 @@
+}
+
+// IDMapMount applies GID/UID shift according to gidmap/uidmap for target path
+func IDMapMount(source, target string, usernsFd int) (err error) {
+	var (
+		attr unix.MountAttr
+	)
+
+	attr.Attr_set = unix.MOUNT_ATTR_IDMAP
+	attr.Attr_clr = 0
+	attr.Propagation = 0
+	attr.Userns_fd = uint64(usernsFd)
```

golangci-lint (1.64.8)

[high] G115: integer overflow conversion int -> uint64

(gosec)